phpunit.xml.dist

```diff
@@ -50,26 +48,38 @@
     <env name="DTT_MINK_DRIVER_ARGS" value='["firefox", null, "http://localhost:4444/wd/hub"]'/>
   </php>
   <testsuites>
+    <testsuite name="unit">
+      <file>src/PhpUnit/JoinupUnitTestSuite.php</file>
+    </testsuite>
+    <testsuite name="kernel">
+      <file>src/PhpUnit/JoinupKernelTestSuite.php</file>
+    </testsuite>
+    <testsuite name="functional">
+      <file>src/PhpUnit/JoinupFunctionalTestSuite.php</file>
+    </testsuite>
+    <testsuite name="functional-javascript">
+      <file>src/PhpUnit/JoinupFunctionalJavascriptTestSuite.php</file>
+    </testsuite>
+    <testsuite name="existing-site">
+      <file>src/PhpUnit/JoinupExistingSiteTestSuite.php</file>
+    </testsuite>
+    <testsuite name="existing-site-javascript">
+      <file>src/PhpUnit/JoinupExistingSiteJavascriptTestSuite.php</file>
+    </testsuite>
   </testsuites>
   <listeners>
     <listener class="\Drupal\Tests\Listeners\DrupalListener">
     </listener>
     <!-- The Symfony deprecation listener has to come after the Drupal listener -->
-    <!--
-      This has been disabled until we have decided on an approach to tackle deprecated code.
-      Ref. https://webgate.ec.europa.eu/CITnet/jira/browse/ISAICP-5208
-      <listener class="Symfony\Bridge\PhpUnit\SymfonyTestsListener">
-      </listener>
-    -->
+    <listener class="Symfony\Bridge\PhpUnit\SymfonyTestsListener">
+    </listener>
   </listeners>
   <!-- Filter for coverage reports. -->
   <filter>
     <whitelist>
-      <directory>./core/includes</directory>
-      <directory>./core/lib</directory>
-      <directory>./core/modules</directory>
-      <directory>./modules</directory>
-      <directory>./sites</directory>
+      <directory>web/modules/custom</directory>
+      <directory>web/profiles</directory>
+      <directory>web/themes</directory>
```

Comment:
I'm curious if this change will help to get the code coverage report finished in <2 hours but I am not counting on it.

Reply:
No, it has no effect :(